app/uk/gov/hmrc/helptosavereminder/actors/ProcessingSupervisor.scala

```diff
@@ -97,7 +97,24 @@ class ProcessingSupervisor @Inject()(
 
       (isUserScheduleEnabled, isExpressionValid) match {
```

Comment:
Surely we want to do this logging regardless of whether scheduling is enabled and regardless of whether the scheduling expression is valid?

This does not do what the story asked:

This code is useless, it will never output anything unless the service is started on the day of the schedule and before the schedule runs, because it is only fetching the records due for processing, not all records.  The description made this clear, saying "These reports should be irrespective of the next scheduled date."

1. Where is the list of list of scheduled reminder times and their frequencies?

2. There was a typo in the story description, it asked for "frequencies or each frequency of email addresses" rather than "frequencies **of** each frequency of email addresses" never the less it made it clear later in the sentence.  You are instead just reporting the number of email addresses.

It would be far preferable if the code was not dependent on what reminder dates can be used, I do not think the service is dependent anywhere else on those - it is the frontend that has those hard coded.  So it would be better to  establish the set of sequences present and count each.

Reply:
Done